README.md

```
@@ -45,11 +45,11 @@ English | [简体中文](README_zh.md)
 ## 🚀 Quick Start
 
 ```bash
-git clone https://github.com/wecode-ai/wegent.git && cd wegent
-docker-compose up -d
-# Open http://localhost:3000
+curl -fsSL https://raw.githubusercontent.com/wecode-ai/Wegent/main/install.sh | bash
 ```
 
+Then open http://localhost:3000 in your browser.
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)